pkgs/by-name/tt/tt/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+}:
+
+buildGoModule rec {
+  pname = "tt";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "lemnos";
+    repo = "tt";
+    rev = "v${version}";
+    hash = "sha256-vKh19xYBeNqvVFilvA7NeQ34RM5VnwDs+Hu/pe3J0y4=";
+  };
+
+  vendorHash = "sha256-edY2CcZXOIed0+7IA8kr4lAfuSJx/nHtmc734XzT4z4=";
+
+  postInstall = ''
+    mv $out/bin/src $out/bin/tt
```

In order to be able to use `installManPage` here the `installShellFiles` input (see [my comment above](https://github.com/NixOS/nixpkgs/pull/308015/files#r1587110881)) is needed. Is this helpful for you @Sail0rd?
```suggestion
    mv $out/bin/src $out/bin/tt
    installManPage tt.1.gz
```

---

Really helpful, thanks for the explanation.